test/LendingPool.t.sol

```diff
@@ -989,26 +1000,25 @@ contract AccountingTest is LendingPoolTest {
     }
 
     function testSuccess_liquidityOf(
```

Comment:
Don't delete fuzzed interest rate!
Instead use cheatcodes to set the interest rate